pkg/agent/endpoints/workload/handler.go

```diff
@@ -145,22 +145,27 @@ func (h *Handler) FetchJWTBundles(req *workload.JWTBundlesRequest, stream worklo
 
 // ValidateJWTSVID processes request for JWT SVID validation
 func (h *Handler) ValidateJWTSVID(ctx context.Context, req *workload.ValidateJWTSVIDRequest) (*workload.ValidateJWTSVIDResponse, error) {
+	log := h.Log.WithField(telemetry.Method, telemetry.ValidateJWTSVID)
 	if req.Audience == "" {
+		log.Error("audience must be specified")
```

Comment:
Logs should start with capitalization but audience refers to an argument, so it would be weird. Maybe

```
log.Errorf("Missing required audience parameter")
```